andela/travel_tool_front
# #160370617 Edit requests with status 'open'

#### What does this PR do?
- Allows open requests to be edited

#### Description of Task to be completed?
- Create component for editing a request
- Modify the table component to include dropdown for editing
- Create sagas, action and reducer for editing open requests
- Write tests for sagas, action, reducer


#### How should this be manually tested?
- Clone the repository
- Pull this branch
- Change into the project directory `cd travel_tool_front`
- Start the server using `make start`
- Start the backend using `make start`
- Open your browser and navigate to `http://travel-tool.andela.com:3000/requests`
- Click on the ellipsis icon for an open request, you should see a dropdown with an option to edit
- Click on the edit button, update the displayed form with the new information and click `Update Request`. The updated request should be displayed on the request table

#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
[#160370617](https://www.pivotaltracker.com/story/show/160370645)


#### Screenshots (if appropriate)


#### Questions:
N/A
